pkg/apis/helm.cattle.io/v1/types.go

```diff
@@ -37,6 +37,7 @@ type HelmChartSpec struct {
 	AuthSecret      *corev1.LocalObjectReference  `json:"authSecret,omitempty"`
 
 	AuthPassCredentials bool `json:"authPassCredentials,omitempty"`
+	DockerRegistrySecret *corev1.LocalObjectReference  `json:"dockerRegistrySecret,omitempty"`
```

Comment:
please make sure that all the files are formatted using `go fmt`. Most editors do this for you but the indentation here appears to be off.